src/Redux/Book/reducer.js

```diff
@@ -33,4 +34,45 @@ const bookReducer = (state = initialState, action) => {
   }
 };
 
+export const getBooksList = () => async (dispatch) => {
+  const booksList = await fetch(`${baseURL}`)
+    .then((response) => response.json());
+  const booksID = Object.keys(booksList);
+  const formatedBooks = [];
+  booksID.map((key) => formatedBooks.push({
+    id: key,
```

Comment:
When I add a book and then refresh the page, the book that I added disappears.

- Kindly make sure that books are being saved and retrieved from the API properly. When I refresh the page, data should not be lost because it is permanently stored in the API.